core/indigo-core/tests/common.cpp

```diff
@@ -75,3 +77,40 @@ std::string IndigoCoreTest::dataPath(const char* dataPathSuffix)
 {
     return dataPathPrefix + "/" + dataPathSuffix;
 }
+
+void IndigoCoreTest::loadReaction(const char* buf, Reaction& reaction)
+{
+    BufferScanner scanner(buf);
+    ReactionAutoLoader loader(scanner);
+    loader.loadReaction(reaction);
+}
+
+void IndigoCoreTest::loadQueryReaction(const char* buf, QueryReaction& queryReaction)
```

Comment:
can we have function overloading with loadReaction?

Reply:
Done